packages/elasticsearch-asset-apis/test/helpers.ts

```diff
@@ -0,0 +1,143 @@
+import {
```

Comment:
I thought we used real elasticsearch now?

Reply:
We do, except for the spaces-reader tests. This fake client is used for that since we don't have a living spaces instance to test against